indico/modules/events/editing/controllers/backend/management.py

```diff
@@ -145,3 +146,25 @@ def _process_POST(self, editable_types):
         editable_types_names = [t.name for t in editable_types]
         editing_settings.set(self.event, 'editable_types', editable_types_names)
         return '', 204
+
+
+class RHEditableTypePrincipals(RHEditingManagementBase):
+    def _process_args(self):
+        RHEditingManagementBase._process_args(self)
+        self.editable_type = EditableType[request.view_args['type']]
+
+    def _process_GET(self):
+        principals = {p.principal for p in self.event.acl_entries
+                      if p.has_management_permission(self.editable_type.name + '_editing', explicit=True)}
+        return EditableTypePrincipalsSchema().jsonify({'principals': principals})
+
+    @use_rh_kwargs(EditableTypePrincipalsSchema)
+    def _process_POST(self, principals):
+        permission_name = self.editable_type.name + '_editing'
```

Comment:
I'd prefer a dict mapping editable types to permission names. Otherwise searching for the permission name won't find this.